bundles/org.eclipse.passage.lbc.api/src/org/eclipse/passage/lbc/internal/api/BackendLicenseLock.java

```diff
@@ -12,22 +12,21 @@
  *******************************************************************************/
 package org.eclipse.passage.lbc.internal.api;
 
-import java.util.function.Supplier;
-
 import org.eclipse.passage.lic.internal.api.ServiceInvocationResult;
+import org.eclipse.passage.lic.internal.api.conditions.Condition;
```

Comment:
does this import beget `restriction` watning?

Reply:
no, *.lbc.base and *.lbc.api are x-friends of *.lic.api